spec/facades/back_end_facade_spec.rb

```diff
@@ -12,7 +12,7 @@
     gyms = BackEndFacade.get_user_gyms(1)
 
     expect(gyms[0]).to be_an_instance_of(UserGym)
-    expect(gyms.count).to eq(4)
+    expect(gyms.count).to eq(3)
```

Comment:
We need to be careful merging this... I cleaned this up, don't want this to get merged back in to hard-coded test values.